_schemaV2.graphql

```diff
@@ -8015,6 +8015,41 @@ enum ConsignmentSubmissionStateAggregation {
   SUBMITTED
 }
 
+type Contact {
+  canContact: Boolean
+  email: String
+
+  # A globally unique ID.
+  id: ID!
+
+  # A type-specific ID.
+  internalID: ID!
+  location: Location
```

Comment:
I was today years old when I learned contacts can have locations 😅

Reply:
Wait 'til you hear about their `Communications` objects

Reply:
please do not look into partner related modeling 😝